oml/configs/datasets/image_labeled_dataset.yaml

```diff
@@ -0,0 +1,9 @@
+name: image_labeled_dataset
+args:
+  dataframe_name: df.csv
```

Comment:
let's use "df" name here? so it's more consistent with argument names, but replace it with read actual df in runtime